apps/ecs/index.js

```diff
@@ -22,7 +23,45 @@ module.exports = {
 
     },
     '/already-employed': {
-
-    }
+      fields: ['person-work-for-you'],
+      forks: [
```

Comment:
In this case, where the answer has only two options, the next property can be used to define the default next step if the condition in the fork is not met. This reduces the amount of conditional checks.